packages/blockchain-extension/extension/commands/packageSmartContractCommand.ts

```diff
@@ -249,9 +301,31 @@ async function javaPackageAndVersion(overrideName?: string, overrideVersion?: st
 /**
  * Method which calls an input box should the project be coded in golang, which asks the user for a package name and version
  * (as golang projects do not contain a package.json file), and returns an object containing both these values.
+ * @param workspaceDir {String} workspaceDir A string containing the path to the current active workspace (the workspace of the project the user is packaging).
  * @returns {string, string} Returns an object with the workspacePackageName and workspacePackageVersion which will be used in the createPackageDir() method
  */
-async function golangPackageAndVersion(overrideName?: string, overrideVersion?: string): Promise<{ workspacePackageName: string, workspacePackageVersion: string }> {
+async function golangPackageAndVersion(workspaceDir: vscode.WorkspaceFolder, overrideName?: string, overrideVersion?: string): Promise<{ workspacePackageName: string, workspacePackageVersion: string }> {
+
+    // Check if go smart contract uses version greater that fabric v2
+
+    const workspaceMod: string = path.join(workspaceDir.uri.fsPath, '/go.mod');
```

Comment:
No need for `/go.mod` - path.join will add slashes depending on the platform, so `go.mod` is fine.